directives/annotated.py

```diff
@@ -87,11 +92,26 @@ def depart_annotated_node(self, node):
     parsed_html = self.body  # extract generated feedback line
     self.body = env.redirect # restore original output
 
-    self.body.append(postprocess(u''.join(parsed_html), node['name']))
+    postprocessed_html = postprocess_annotation_tags(u''.join(parsed_html), node['name'])
```

Comment:
Unicode literals are not needed anymore since we only use Python 3: `u''` -> `''` (remove u)

Reply:
OK. I didn't actually edit that call in this PR, but I fixed that line now. At some point, we could remove all the unnecessary u's from the repo.